core/types/receipt.go

```diff
@@ -95,7 +95,7 @@ type receiptRLP struct {
 }
 
 // storedReceiptRLP is the storage encoding of a receipt.
-type storedReceiptRLP struct {
+type StoredReceiptRLP struct {
```

Comment:
Per [this comment](https://github.com/ethereum/go-ethereum/pull/26225/files#r1030844974) from the original PR, I don't think we want to expose this in our public API.

Reply:
would renaming help in avoiding any further confusions? or should we let it be?